packages/core/src/builder.class.ts

```diff
@@ -2634,7 +2634,6 @@ export class Builder {
     const isApiCallForCodegenOrQuery = isApiCallForCodegen || this.apiEndpoint === 'query';
 
     if (this.apiEndpoint === 'content') {
-      queryParams.enrich = true;
```

Comment:
Why are we removing this arg?

Reply:
Here is a loom explaining why i removed that line - https://www.loom.com/share/89fbde11974a43839537868e59708e1d